libshaderc_util/src/compiler_test.cc

```diff
@@ -221,4 +221,40 @@ TEST_F(CompilerTest, AddMacroDefinitionNotNullTerminated) {
   EXPECT_TRUE(SimpleCompilationSucceeds(kMinimalExpandedShader, EShLangVertex));
 }
 
+// A convert-string-to-vector test case consists of 1) an input string; 2) an
+// expected vector after the conversion.
+struct ConvertStringToVectorTestCase {
+  const std::string str_;
```

Comment:
The style guide says that struct member names don't end with underscore.
Only class member names do.